src/gui/tray/ActivityItem.qml

```diff
@@ -128,13 +128,13 @@ MouseArea {
                     
                     imageSource: !primary ? activityActionsLayout.actionButtonIcon(actionIndex) : ""
                     
-                    textColor: primary ? Style.ncBlue : "black"
+                    textColor: primary ? UserModel.currentUser.accentColor : "black"
                     textColorHovered: Style.lightHover
                     
-                    textBorderColor: Style.ncBlue
+                    textBorderColor: UserModel.currentUser.accentColor
                     
                     textBgColor: "transparent"
-                    textBgColorHovered: Style.ncBlue
+                    textBgColorHovered: UserModel.currentUser.accentColor
```

Comment:
@claucambra So seems you are gonna need to rebase against master and adjust to changes that I've made for the new Activity Buttons style. e.g. this `textBgColorHovered` is no longer there, and, yo will also need to take care of button background to make sure the text is visible on those when its color is changed

Reply:
Rebased successfully!